support/build/Makefile.build

```diff
@@ -53,9 +55,11 @@ $(foreach S,$($(call uc,$(L))_SRCS) $($(call uc,$(L))_SRCS-y) \
 	    $(EACHOLIB_SRCS) $(EACHOLIB_SRCS-y), \
 $(eval $(call buildrule_libobj_multitarget,$(L),$(S))) \
 ); \
-$(if $(call qstrip,$($(call uc,$(L))_OBJS) $($(call uc,$(L))_OBJS-y)), \
+$(if $(call qstrip,$($(call uc,$(L))_OBJS) $($(call uc,$(L))_OBJS-y) \
+                   $($(call uc,$(L))_ALIBS) $($(call uc,$(L))_ALIBS-y)), \
 $(eval $(call buildrule_olib,$(L))); \
 $(eval $(call uc,$(P))_OLIBS-y    += $(call libname2olib,$(L))); \
+$(eval UK_ALIBS-y += $($(call uc,$(L))_ALIBS) $($(call uc,$(L))_ALIBS-y)); \
```

Comment:
Hum... I am not sure if this makes so much sense. If you want to link with a static library at the last step, you would register such a lib to `UK_ALIBS-y` in your `Makefile.uk` (like with `UK_ALIBS-$(CONFIG_MYLIB)+=mylib.a`). Similar to other scopes, it was intended that static libraries that are registered to a library focus are linked before the `exportsyms.uk` is applied. How useful this feature is, i a different question because linking with static libraries work differently (simple form of DCE).

Reply:
`UK_ALIBS-$(CONFIG_MYLIB)+=mylib.a)` doing this right now does not work as expected. The alib is imidiatly link to an object file.

Reply:
Actually you're right, this exact line doesn't make sense.